src/web-components/comments/components/content.ts

```diff
@@ -111,20 +110,22 @@ export class CommentsContent extends WebComponentsBaseElement {
   };
 
   protected render() {
-    return html` ${repeat(
-      this.annotations.filter((annotation) => annotation.comments?.length),
-      (annotation: Annotation) => annotation.uuid,
-      (annotation: Annotation) => html`
-        <superviz-comments-annotation-item
-          annotation=${JSON.stringify(annotation)}
-          selected="${this.selectedAnnotation}"
-          annotationFilter=${this.annotationFilter}
-          uuid=${annotation.uuid}
-          isLastComment=${this.checkLastAnnotation(annotation.uuid)}
-          participantsList=${JSON.stringify(this.participantsList)}
-        >
-        </superviz-comments-annotation-item>
-      `,
-    )}`;
+    return html`
+      ${this.annotations
+        .filter((annotation) => annotation.comments?.length)
+        .map(
+          (annotation: Annotation) => html`
+            <superviz-comments-annotation-item
+              annotation=${JSON.stringify(annotation)}
+              selected="${this.selectedAnnotation}"
+              annotationFilter=${this.annotationFilter}
+              uuid=${annotation.uuid}
+              isLastComment=${this.checkLastAnnotation(annotation.uuid)}
+              participantsList=${JSON.stringify(this.participantsList)}
+            >
+            </superviz-comments-annotation-item>
+          `,
+        )}
+    `;
```

Comment:
Use repeat instead map because it's prevent weird behaviors. 

See more here: https://lit.dev/docs/templates/lists/#when-to-use-map-or-repeat